scalding/src/main/scala/com/matthewrathbone/scalding/Main.scala

```diff
@@ -0,0 +1,35 @@
+package main.scala.com.matthewrathbone.scalding
+
+import com.twitter.scalding._
+import cascading.tuple.TupleEntry
+
+class Main ( args: Args ) extends Job( args )
+  {
+  
+    val inputFields = 'line
+	val users = ( 'id, 'email, 'language, 'location)
+  	val transactions = ( 'transaction_id, 'product_id, 'user_id, 'purchase_amount, 'item_description);
+	
+	val input1 = TextLine( args( "input1" ) )
+	val input2 = TextLine( args( "input2" ) )
+	val output = Tsv( args( "output" ) )
+	
+	val usersInput = input1.read.mapTo( inputFields -> users )
+    { te: TupleEntry =>
```

Comment:
Scala coding convention is that open curly follows the first set of parens, like this:

```
val a = someFunction("hello") { world =>
  (world, "is awesome")
}
```